examples/react/multi-file-upload/.gitignore

```diff
@@ -1,2 +1,24 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
 node_modules
-build
```

Comment:
some folks might have an old `build` so lets leave this entry in the .gitignore for now